src/sync_template.sh

```diff
@@ -293,19 +293,19 @@ function push () {
   local is_force=$2
   local is_with_tags=$3
 
-  local additional_params=" "
+  args=(--set-upstream origin "${branch}")
 
   if [ "$is_force" == true ] ; then
     warn "forcing the push."
-    additional_params="${additional_params}--force "
+    args+=(--force)
```

Comment:
thanks a bunch for the PR! :rocket: 
just thought about the order of the arguments for `git push`.
Like `args+=(--force args[@])`

It definititly works with your change. Was just out of a convenience part.
But let's keep it like that :ninja: